test/tools/ossfuzz/yulFuzzerCommon.cpp

```diff
@@ -20,11 +20,13 @@ using namespace std;
 using namespace yul;
 using namespace yul::test::yul_fuzzer;
 
-void yulFuzzerUtil::interpret(ostream& _os, shared_ptr<yul::Block> _ast)
+void yulFuzzerUtil::interpret(ostream& _os, shared_ptr<yul::Block> _ast, size_t _maxSteps, size_t _maxTrace,
```

Comment:
Usually the coding still would be:
```suggestion
void yulFuzzerUtil::interpret(
	ostream& _os,
	shared_ptr<yul::Block> _ast,
	size_t _maxSteps,
	size_t _maxTrace,
	size_t _maxMemory
)
```
Same for the declaration and the calls.

Reply:
Thank you @ekpyron . Fixed the code style. Hope I got it right :)